core/src/main/java/io/micronaut/core/io/scan/DefaultClassPathResourceLoader.java

```diff
@@ -40,8 +40,6 @@
  */
 public class DefaultClassPathResourceLoader implements ClassPathResourceLoader {
 
-    private static final Logger LOG = LoggerFactory.getLogger(DefaultClassPathResourceLoader.class);
```

Comment:
what is the motivation of this change?

Reply:
Probably to avoid native compilation time processing

Reply:
To avoid the logs being initialized at build time